src/VisualStudio/IntegrationTest/New.IntegrationTests/VisualBasic/BasicErrorListCommon.cs

```diff
@@ -109,7 +109,7 @@ End Namespace
                 string.Join(Environment.NewLine, actualContents));
         }
 
-        [IdeFact, WorkItem("https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1643350")]
+        [IdeFact(Skip = "https://github.com/dotnet/roslyn/issues/72428"), WorkItem("https://dev.azure.com/devdiv/DevDiv/_workitems/edit/1643350")]
```

Comment:
Is this test skipped because of the "tiny time slice" while things are loading? If so, seems it would be better to set the feature flag in the registry before running VS for the tests, to ensure not delay waiting for TN.